### PR TITLE
chore(actions): extract label values and utils to support module

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** N/A

## Summary

✨🧹✨

### CODE REVIEW

**Review by geospatialem — APPROVED** [review]
Awesome refactor! 🧹

LGTM! 💪🏻

File: .github/scripts/support/labels.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const labels = {
```
Comment (benelan) [inline .github/scripts/support/labels.js:1]: Nitpick: should we call this file `resources.js` so we can add additional objects in here like we do with the components?

**Review by benelan — APPROVED** [review]
Nice cleanup!

File: .github/scripts/support/labels.js (hunk 1 above)
Comment (jcfranco) [reply .github/scripts/support/labels.js:1]: Good idea. Will rename!

**Pull request merged**